src/scene/castlescenecore.pas

```diff
@@ -437,6 +437,9 @@   TPlayAnimationParameters = class
     if the scene does not change (in a significant way) between each query.
     So you can query information like @link(LocalBoundingBox),
     @link(BoundingBox), @link(VerticesCount)... as often as you want to. }
+
+  { TCastleSceneCore }
```

Comment:
Do not add such comments with just class name. They are a bit useless (a person reading doc, or source code, will see the class name anyway), and would obscure the good documentation string for PasDoc. Lazarus adds them automatically, I recommend to disable it in Lazarus settings, see the screenshot in https://github.com/pasdoc/pasdoc/issues/149#issuecomment-1308837496 .